concepts/05 UI Components/Toast/00 Getting Started with Toast/05 Display a Toast/05 Use the 'notify' Method.md

```diff
@@ -0,0 +1,265 @@
+Call the **notify** method to display a Toast. This method can accept two slightly different sets of arguments. 
+
+A simpler syntax is [notify(message, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_type_displayTime). You specify the [message](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#message), [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type), and [displayTime](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#displayTime).
```

Comment:
Simpler compared to what?

Reply:
Would it be okay if I change "simpler" to "basic"?
The thing is you can use the first notify method when you do not need the additional customization of the toast. So it's a basic configuration.

Reply:
Ok